config/locales/defaults/en.yml

```diff
@@ -0,0 +1,10 @@
+en:
```

Comment:
De acordo com https://guides.rubyonrails.org/i18n.html#organization-of-locale-files nos temos um padrao para as views tambem. No caso essas informacoes aqui ficariam dentro de uma pasta views, ai o general ficaria dentro de default e o store dentro de store.

Reply:
Ah e no caso das views elas sempre ficam dentro de uma key views:

```
en:
  views:
    ....
```